src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CreateClient.cs

```diff
@@ -17,6 +17,12 @@ public static void NullPipeName_Throws_ArgumentNullException()
         {
             AssertExtensions.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(null));
             AssertExtensions.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(".", null));
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Since AccessRights are specific to Windows, could you please move the PlatformSpecific tests to a separate file? It could be named NamedPipeTest.CreateClient.Windows.cs and can be placed in the same csproj ItemGroup as the other Windows-specific tests.

Reply:
I was torn on this and saw trade-offs in either direction (separate file for improved cleanliness, or same file for improved discoverability). But as you call out, there is already a heavy usage of platform-specific files, so I'll make the change.